doc/tor.md

```diff
@@ -41,9 +41,11 @@ outgoing connections, but more is possible.
     -onion=ip:port  Set the proxy server to use for Tor onion services. You do not
                     need to set this if it's the same as -proxy. You can use -onion=0
                     to explicitly disable access to onion services.
-                    Note: Only the -proxy option sets the proxy for DNS requests;
+                    ------------------------------------------------------------------
+                    <b>Note: Only the -proxy option sets the proxy for DNS requests;
                     with -onion they will not route over Tor, so use -proxy if you
-                    have privacy concerns.
+                    have privacy concerns.</b>
+                    ------------------------------------------------------------------
```

Comment:
Maybe just bold or just the horizontal separators.  Adding emphasis is noisy and eventually, as more is added for well-intentioned handholding, it makes the doc harder to read and understand and has to be tuned out by readers.  A clean doc is a nice thing.

Reply:
Okay. Will remove bold.

Reply:
Thanks!